neighbors/src/helpers/apiCommunication.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/react-in-jsx-scope */
+import { collection, onSnapshot, doc } from 'firebase/firestore';
+import { useEffect, useState, createContext } from 'react';
+import { PropTypes } from 'prop-types';
+
+import { db } from '../api/firebase';
+export const usersData = collection(db, 'Users');
+export const needHelpPostsData = collection(db, 'Posts-need-help');
+export const offerHelpPostsData = collection(db, 'Posts-offer-help');
+
+export const ThemeContext = createContext();
+export const ThemeProvider = ({ children }) => {
```

Comment:
Jeżeli jest to context od usera to lepiej byłoby go nazwać np. UserContext

Reply:
Poprawione w Branchy o autoryzacji,